README.md

```diff
@@ -205,9 +205,42 @@ packages on the destination side can be ignored with
 
 By default, all subpackages of the specified packages are also ignored, but
 this can be disabled on an individual basis by adding
-`<ignoreSubpackages>false</ignoreSubpackages>` to the `<ignoreSourcePackage>`
+`<filterSubpackages>false</filterSubpackages>` to the `<ignoreSourcePackage>`
```

Comment:
maybe worth adding a brief note below this that the param used to be called `ignoreSubpackages`, its been renamed, but old name is supported as well

... and I just remembered this repo has a CHANGELOG.md that would be good to update too, both about the new functionality and this change

Reply:
great idea! updated both